packages/suite/src/services/coinjoin/config.ts

```diff
@@ -149,7 +149,9 @@ export const getCoinjoinConfig = (
     environment?: CoinjoinServerEnvironment,
 ) => {
     const config = COINJOIN_NETWORKS[network];
-    const settings = config ? config[environment ?? 'public'] : undefined;
+    const settings = config
+        ? config[environment ?? (Object.keys(config)[0] as CoinjoinServerEnvironment)]
```

Comment:
would be nice to add some comment to the config, that `public` has to be as first at least for mainnet?

Reply:
commented on each network